src/main/java/dev/amble/ait/core/handles/HandlesResponse.java

```diff
@@ -69,6 +69,34 @@ default boolean isCommand(String command) {
         return getCommandWords().contains(command.toLowerCase());
     }
 
+    // i have no idea how this shit fucking works
+    default int distance(String command, String input) {
+        int[][] dp = new int[command.length() + 1][input.length() + 1];
```

Comment:
Wouldn't it be enough to initiate the array with the exact lengths instead of length + 1?
Because nowhere in the method does it ever access the length + 1 element.

For example, if command was `refuel`, which has a length of 6, then the loop in the method would iterate from 1 to 6 (both including).
Since we never access index 7 (which is length + 1) anywhere, it should suffice to use only the length to initialize the array (for both command and input).

```suggestion
        int[][] dp = new int[command.length()][input.length()];
```